core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/runtimeModels.scala

```diff
@@ -78,6 +78,7 @@ object RuntimeStatus extends Enum[RuntimeStatus] {
 
   // NOTE: Remember to update the definition of this enum in Swagger when you add new ones
   // transition statuses
+  case object PreCreating extends RuntimeStatus //This status exists when we saves a runtime after createRuntime request, but we haven't published the even to back leo
```

Comment:
Maybe `Provisioning`? 🤔 

I like this idea of this status but we should double-check with API consumers (Terra UI, AoU). Our status codes are a contract that we provide and we should make sure adding a new one doesn't break anything.

Reply:
I didn't use `Provisioning` just because gce is already using the word.

For contract, I think client doesn't need to know this status, we can map it to `Creating` in contract

Reply:
👍 to map it to `Creating` in contract